source/_components/device_tracker.gpslogger.markdown

```diff
@@ -11,63 +11,8 @@ ha_category: Presence Detection
 ha_release: 0.34
 ---
```

Comment:
We should change the title of this page to distinguish it from component. Right now that's impossible in the component search list in the docs.

Reply:
Fixed.